app/models/vm_question_response.rb

```diff
@@ -161,14 +161,33 @@ def add_answer(answer)
     end
   end
 
+  def calculate_metrics
```

Comment:
Need to add comments for this in the upcoming commits

Reply:
Add comments for newly added method